### PR TITLE
Enrich ecs:DescribeTasks responses.

### PR DESCRIPTION
**Pull request by kkuzmin**
https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_DescribeTasks.html

### CODE REVIEW

**Review by motobob — APPROVED** [review]
LGTM

File: include/erlcloud_ecs.hrl [hunk 1]
```diff
@@ -247,21 +259,51 @@
...
+    desired_status :: undefined | binary(),
+    group :: undefined | binary(),
+    health_status :: undefined | binary(),
     last_status :: undefined | binary(),
     launch_type :: undefined | binary(),
+    memory :: undefined | binary(),
     overrides :: undefined | #ecs_task_override{},
-    started_at :: undefined | pos_integer(),
+    platform_version :: undefined | binary(),
+    pull_started_at :: undefined | number(),
```
Comment (nalundgaard) [inline include/erlcloud_ecs.hrl:297]: Is this one actually a `binary()`? It doesn't look like a number in the [docs](https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_Task.html#ECS-Type-Task-pullStartedAt) or test sample.

File: include/erlcloud_ecs.hrl [hunk 2]
```diff
@@ -247,21 +259,51 @@
...
+    pull_stopped_at :: undefined | number(),
```
Comment (nalundgaard) [inline include/erlcloud_ecs.hrl:298]: Is this one actually a `binary()`? It doesn't look like a number in the [docs](https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_Task.html#ECS-Type-Task-pullStoppedAt) or test sample.

File: include/erlcloud_ecs.hrl [hunk 3]
```diff
@@ -247,21 +259,51 @@
...
+    started_at :: undefined | number(),
```
Comment (nalundgaard) [inline include/erlcloud_ecs.hrl:299]: Is this one actually a `binary()`? It doesn't look like a number in the [docs](https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_Task.html#ECS-Type-Task-startedAt) or test sample.

File: include/erlcloud_ecs.hrl [hunk 4]
```diff
@@ -247,21 +259,51 @@
...
     started_by :: undefined | binary(),
-    stopped_at :: undefined | pos_integer(),
+    stopped_at :: undefined | number(),
```
Comment (nalundgaard) [inline include/erlcloud_ecs.hrl:301]: Is this one actually a `binary()`? It's a Timestamp in the [docs](https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_Task.html#ECS-Type-Task-stoppedAt).

File: include/erlcloud_ecs.hrl [hunk 5]
```diff
@@ -226,14 +226,26 @@
...
+-record(ecs_network_interface, {
+    attachment_id :: undefined | binary(),
+    private_ipv4_address :: undefined | binary()
+}).
+
 -record(ecs_container, {
     container_arn :: undefined | binary(),
-    exit_code :: undefined | pos_integer(),
+    cpu :: undefined | binary(),
+    exit_code :: undefined | binary(),
```
Comment (nalundgaard) [inline include/erlcloud_ecs.hrl:237]: Is this a `binary()`? looks like an integer [here](https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_Container.html#ECS-Type-Container-exitCode).

File: include/erlcloud_ecs.hrl [hunk 6]
```diff
@@ -247,21 +259,51 @@
...
+    key :: undefined | binary(),
+    value :: undefined | binary()
+}).
+
 -record(ecs_task, {
+    attachments :: undefined | [#ecs_attachment{}],
+    availability_zone :: undefined | binary(),
     cluster_arn :: undefined | binary(),
+    connectivity :: undefined | binary(),
+    connectivity_at :: undefined | number(),
```
Comment (nalundgaard) [inline include/erlcloud_ecs.hrl:284]: This is a [Timestamp](https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_Task.html#ECS-Type-Task-connectivityAt), I assume that's actually... a `binary()` for us? Example says so.

File: include/erlcloud_ecs.hrl [hunk 7]
```diff
@@ -247,21 +259,51 @@
...
+    started_at :: undefined | number(),
     started_by :: undefined | binary(),
-    stopped_at :: undefined | pos_integer(),
+    stopped_at :: undefined | number(),
     stopped_reason :: undefined | binary(),
+    tags :: undefined | [#ecs_tag{}],
     task_arn :: undefined | binary(),
-    task_definition_arn :: undefined | binary()
+    task_definition_arn :: undefined | binary(),
+    version :: undefined | pos_integer()
```
Comment (nalundgaard) [inline include/erlcloud_ecs.hrl:306]: This one is a `Long` in the [docs](https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_Task.html#ECS-Type-Task-version). Is it right that it's an integer, always positive?

**Review by nalundgaard — CHANGES_REQUESTED** [review]
Looks like mainly spec issues, other stuff LGTM.

**Comment by kkuzmin** [comment]
@nalundgaard addressed. Timestamps are numbers. Fixed the test

**Comment by nalundgaard** [comment]
@kkuzmin thanks. what about the question on version? is it always a `pos_integer()`?

File: include/erlcloud_ecs.hrl (hunk 7 above)
Comment (kkuzmin) [reply include/erlcloud_ecs.hrl:306]: I haven't seen negative version in real life. And doc says

> Every time a task experiences a change that triggers a CloudWatch event, the version counter is incremented

I think it is safe to assume it is non negative

**Comment by kkuzmin** [comment]
@nalundgaard replied to your comment in the review about version type

**Review by nalundgaard — APPROVED** [review]

**Pull request merged**